### DISCUSSION TITLE
Making website more accessible by URLs

### DISCUSSION THREAD

**Discussion by standev** [DivinumOfficium/divinum-officium#2537] [Ideas]
Laudetur Iesus Christus!

It is really awesome that this project is open source and accessible on github. 👍 Here's an idea of improvement. At the moment, the website is built around a CGI script which gives out the proper texts. Which works fine, but makes it impossible for search engines to index the content of the website.

However if website were changed to use different URLs like `divinumofficium.com/la/en/missa/1910/hodie` (similar to how aquinas.cc website is organized) then search engines and Web Archive could index it, save copies, etc; also visitors could be use bookmarks for their favourite setup and then go directly to the needed page. Whereas now it is impossible: one needs to make user choices each time to access the needed page. 

What do you think?

**Comment by mbab** [comment]
Idea is good and proper, but at current stage is rather impossible, first we need to have clean code, then rewrite it to conform contemporary standards.

On other side it can be probably done with mod_rewrite.

**Reply by ajmalton** [reply]
It would be interesting to do a mock-up and see what it looks like in the elegant aquinas.cc style.  I like the way they've split the Options and the Navigation into a hamburger menu and a responsive popup.

For example take a slice of our content: Latin+Czech, 1960 rubrics, two days around next Sunday (such as would be downloaded by `divinum-get --version=1960 --pray=SanctaMissa --from=9-11-2021 --to=9-13-2021 --cgi=lang2=Cesky`).  And see if it can be rearranged to look more responsive and mobile...
